### ISSUE TITLE
An npm install throws a WARN error about a missing Readme

### ISSUE THREAD

**Issue by acorncom** [jonathanKingston/ember-cli-sri#3]
When running an `npm install` (Ember-CLI 1.13.8), I get the following error:

`npm WARN package.json ember-cli-sri@1.0.3 No README data`

  From what I've been able to tell, it looks like the Readme for this project is "missing" because something in it runs afoul of the NPM markdown parser config.  Specifically, I think it might be Lines 76-68 (with example `<script>` tags).

Would using HTML entities for the initial `<script` portion fix the issue?

Background info on the NPM markdown parser is available here:
http://blog.npmjs.org/post/109508231330/nicely-presented-markup

And their markdown sanitizer config is here:
https://github.com/npm/marky-markdown/blob/master/lib/sanitize.js

**Comment by jonathanKingston** [comment]
Can I ask what version of NPM you are running too?

It shouldn't really be those as they are all code blocks. Anyway I will look into this thanks.

**Comment by jonathanKingston** [comment]
```
jonathan@kingston-laptop:~$ npm install ember-cli-sri
ember-cli-sri@1.0.3 node_modules/ember-cli-sri
├── broccoli-sri-hash@1.0.4 (sri-toolbox@0.2.0, broccoli-filter@1.2.2, broccoli@0.16.7)
└── ember-cli-babel@5.1.5 (clone@1.0.2, ember-cli-version-checker@1.1.4, resolve@1.1.6, broccoli-funnel@0.2.8, broccoli-babel-transpiler@5.4.5)
jonathan@kingston-laptop:~$ npm -v
2.14.1
```

**Comment by acorncom** [comment]
I'm running NPM 2.7.4 on 10.10.3
```
dbaker@MBP:~$ npm -v
2.7.4
```

Sadly, I can't reproduce the problem at this point (same machine, 6 hrs later, no other updates installed :-/)  Sorry about the noise.  I'll close this for now and will re-open if it re-occurs.

**Comment by jonathanKingston** [comment]
Cool thanks for letting me know and also closing too.

I was on the same version and updated. I just released a new version 1.0.4.

Thanks again

**Comment by acorncom** [comment]
Appreciate all your hard on this, I've been following developments since I saw you and Stef working on it.
